src/components/Settings/SettingsPage.vue

```diff
@@ -455,8 +457,8 @@ export default {
         return this.addNetwork({ privateKeys }).then(() => {
           this.$message.success(`You successfuly registered in ${network === WalletTypes.BTC ? 'BTC' : 'ETH'} network!`)
           this.updateAccount()
-        }).catch(() => {
-          this.$message.error(`Something was wrong. You didn't register in network`)
+        }).catch((err) => {
```

Comment:
In a lot of places we're don't use brackets for one argument. Maybe better to remove them?